src/main/java/seedu/address/model/person/Office.java

```diff
@@ -7,7 +7,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Office {
 
     public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
```

Comment:
make sure to change any related areas as well if you change name of the class...don't leave any remnants linking to nonexistent old code

Reply:
there is error because the parser parts use the address class, should I also remove that or let gerald handle it later? Because if I just delete not sure if anything bad will happen.

Reply:
just make the changes first so the code works, and maybe add comments if necessary.
anything i will look through it later on

Reply:
I just committed after removing all the address stuff, can see again, it can run when I run Main.main